delivery_postlogistics/models/stock_package_type.py

```diff
@@ -4,8 +4,8 @@
 from odoo import fields, models
 
 
-class ProductPackaging(models.Model):
-    _inherit = "product.packaging"
+class PackageType(models.Model):
+    _inherit = "stock.package.type"
```

Comment:
shouldn't we have a migration step to migrate current data from packaging to package type?

Reply:
`package_carrier_type` is an Odoo native field from Odoo 15, I don't think we need to migrate data